components/FlexibleHeader/src/MDCFlexibleHeaderView.m

```diff
@@ -452,18 +444,11 @@ - (void)safeAreaInsetsDidChange {
       // status bar visibility then we ignore this safeAreaInsetsDidChange event.
       return;
     }
-    CGFloat safeAreaTop = MDCDeviceTopSafeAreaInset();
-    if (_currentSafeAreaInsetTop == safeAreaTop) {
-      return;
-    }
-    _currentSafeAreaInsetTop = safeAreaTop;
 
     // If the min or max height have been explicitly set, don't adjust anything if the values
     // already include a Safe Area inset.
     BOOL hasSetMinOrMaxHeight = _hasExplicitlySetMinHeight || _hasExplicitlySetMaxHeight;
-    if (hasSetMinOrMaxHeight && _minMaxHeightIncludesSafeArea) {
```

Comment:
But we don't want to do any of this when min or max height has been explicitly set and _minMaxHeightIncludesSafeArea hasn't been set to NO :(